### PR TITLE
windows build fixed

### PR DESCRIPTION
**Pull request by Pandoriaantje**
added some fixes to build on windows platform, using Botan >= 3.4 (tested with latest Botan 3.5).
Builds fine with QT 6.7.3 MingGW 11.2.0 and above.

### CODE REVIEW

File: Velocity/about.cpp [hunk 1]
```diff
@@ -9,22 +9,6 @@ About::About(QWidget *parent) :
     setFixedSize(sizeHint());
     setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
     QtHelpers::GenAdjustWidgetAppearanceToOS(this);
-
-#ifdef _WIN32
-    if (!QFile::exists(QtHelpers::ExecutingDirectory() + "Developers.mp3"))
```
Comment (hetelek) [inline Velocity/about.cpp:14]: Why remove this? Would prefer to keep it, it's a nice easter egg

File: Velocity/mainwindow.cpp [hunk 2]
```diff
@@ -419,7 +413,10 @@ void MainWindow::LoadFiles(QList<QUrl> &filePaths)
                             else
                             {
                                 package->Rehash();
-                                package->Resign(QtHelpers::GetKVPath(package->metaData->certificate.ownerConsoleType, this));
+                                if (fileMagic == CON)
+                                {
+                                    package->Resign(QtHelpers::GetKVPath(package->metaData->certificate.ownerConsoleType, this));
```
Comment (hetelek) [inline Velocity/mainwindow.cpp:418]: Curious what effect this was having previously? Was it corrupting some files?

File: Velocity/about.cpp (hunk 1 above)
Comment (hetelek) [reply Velocity/about.cpp:14]: I'm generally in favor of removing dependencies if this allows us to remove Phonon. Maybe we can use `QMediaPlayer` to play it?

**Comment by DerfJagged** [comment]
@Pandoriaantje would you mind sharing the binaries?

**Comment by hetelek** [comment]
Have not tested this much, but I was at least able to build and run on Windows. I will merge this and we can follow up with more PRs if necessary. Thank you @Pandoriaantje !

**Comment by hetelek** [comment]
> @Pandoriaantje would you mind sharing the binaries?

@DerfJagged I just uploaded Windows binaries at https://github.com/hetelek/Velocity/releases/tag/v1.0.0

**Pull request merged**